src/bitcoin-cli.cpp

```diff
@@ -349,6 +350,62 @@ class NetinfoRequestHandler : public BaseRequestHandler
         const double milliseconds{round(1000 * seconds)};
         return milliseconds > 999999 ? "-" : ToString(milliseconds);
     }
+    const UniValue NetinfoHelp()
+    {
+        return std::string{
+            "-netinfo level \"help\" \n\n"
+            "Returns a network peer connections dashboard with information from the remote server.\n"
+            "Under the hood, -netinfo fetches the data by calling getpeerinfo and getnetworkinfo.\n"
+            "An optional integer argument from 0 to 4 can be passed for different peers listings.\n"
+            "Pass \"help\" to see this detailed help documentation.\n"
+            "If more than one argument is passed, only the first one is read and parsed.\n"
+            "Suggestion: use with the Linux watch command for a live dashboard; see example below.\n\n"
```

Comment:
nit, feel free to ignore:
Could use a common Unix/Linux syntax here, putting the manpage section in parantheses:
```suggestion
            "Suggestion: use with the Linux watch(1) command for a live dashboard; see example below.\n\n"
```

Reply:
good idea; done